datadog/dogstatsd/base.py

```diff
@@ -152,7 +152,8 @@ def __enter__(self):
 
         def __exit__(self, type, value, traceback):
             # Report the elapsed time of the context manager.
-            self.statsd.timing(self.metric, time() - self.start,
+            time_ms = int(round(1000 * (time() - self.start)))
```

Comment:
Well.. I think it's a good idea to be explicit when converting from floats to integers with what to do with the decimal points.  The choices are round, floor or ceil.  rounding would yield the most accurate long-term data.

However, we're talking about fractions of milliseconds--it feels like it wouldn't matter either way.